### PR TITLE
Switch from LightDM to SDDM

### PR DESCRIPTION
**Pull request by m-wynn**
There's still some nice things that could be done with the theme, but
this works.  It's certainly better than lightdm-webkit-greeter.

### CODE REVIEW

**Comment by m-wynn** [comment]
I tested this on Niblick, but did not wipe beforehand.  IDK how to wipe without it first-booting into an ansible-pull from master, which would install lightdm and fail this role.

**Comment by kulinacs** [comment]
We can test from pxe tonight.

File: roles/cv-greeter-theme/tasks/main.yml [hunk 1]
```diff
@@ -1,45 +1,48 @@
...
-      - ImageMagick
-    state: present
-
-- name: Deploy cv-greeter-theme
+- name: Copy theme
   git:
-    repo: https://github.com/collegiumv/cv_greeter_theme.git
-    dest: /usr/share/lightdm-webkit/themes/cv
-    force: true
+    repo: git://github.com/m-wynn/sddm_wynn-theme.git
```
Comment (kulinacs) [inline roles/cv-greeter-theme/tasks/main.yml]: I'd like to see this forked to CV

**Review by kulinacs — CHANGES_REQUESTED** [review]
Thanks for getting this done. I'm excited to move off lightdm.

File: roles/cv-greeter-theme/tasks/main.yml (hunk 1 above)
Comment (m-wynn) [reply roles/cv-greeter-theme/tasks/main.yml]: Forked and gave System Administrators full access

File: roles/cv-greeter-theme/tasks/main.yml [hunk 2]
```diff
@@ -1,45 +1,48 @@
...
-- name: Install Greeter Dependencies
-  xbps:
-    pkg:
-      - git-all
-      - ImageMagick
-    state: present
-
-- name: Deploy cv-greeter-theme
+- name: Copy theme
   git:
```
Comment (phy1729) [inline roles/cv-greeter-theme/tasks/main.yml]: Perhaps `depth=1` here?

File: group_vars/workstation.yml [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+  accent2_hover: '#009688'
+  default_session: 'cinnamon'
+  username_entry: 'fill'
+  wallpaper: 'https://verdict4u.files.wordpress.com/2016/09/google-now-wallpaper-1.png'
+  session_api: 'http://ldap-api.collegiumv.org/%s/desktopEnvironment/0'
+  aup: |
+    [1] Unauthorized use is prohibited;
+    [2] Usage may be subject to security testing and monitoring;
+    [3] Misuse is subject to criminal prosecution; and
+    [4] Users have no expectation of privacy except as otherwise provided by applicable privacy laws.
```
Comment (dinoocch) [inline group_vars/workstation.yml]: Line too long 😞 

Could we increase the limit to 160 characters?

**Review by dinoocch — APPROVED** [review]

File: group_vars/workstation.yml (hunk 3 above)
Comment (kulinacs) [reply group_vars/workstation.yml]: I'm fine with an increase.

**Pull request merged**